commit cbf984531426e9bd533dbae9dd15944271c1012e
Author: Volker Krause <vkrause@kde.org>
Date:   Mon Nov 17 17:28:05 2025 +0100

    Fix use as static library with Qt 6.10
    
    In a static build private link dependencies become public, and with Qt 6.10
    we need to explicitly search for private Qt libraries.

diff --git a/cmake/QuotientConfig.cmake.in b/cmake/QuotientConfig.cmake.in
index ac219576..5b8b5a75 100644
--- a/cmake/QuotientConfig.cmake.in
+++ b/cmake/QuotientConfig.cmake.in
@@ -11,6 +11,10 @@ find_dependency(Olm)
 find_dependency(OpenSSL)
 find_dependency(@Qt@Sql)
 
+if(NOT @BUILD_SHARED_LIBS@ AND @Qt@Core_VERSION VERSION_GREATER_EQUAL 6.10)
+    find_dependency(@Qt@CorePrivate)
+endif()
+
 include("${CMAKE_CURRENT_LIST_DIR}/@QUOTIENT_LIB_NAME@Targets.cmake")
 
 if (NOT QUOTIENT_FORCE_NAMESPACED_INCLUDES)
